src/NzbDrone.Core/MediaCover/MediaCoverService.cs

```diff
@@ -88,6 +93,11 @@ private void EnsureCovers(Series series)
                     if (!_coverExistsSpecification.AlreadyExists(cover.Url, fileName))
                     {
                         DownloadCover(series, cover);
+                        EnsureResizedCovers(series, cover, true);
+                    }
+                    else
```

Comment:
I know what you mean, but you fail to see the parameters are different :) The one after DownloadCover supplies 'true' which forces the resized images to be recreated even if they already exist.